templates/definition/charger/ocpp.yaml

```diff
@@ -0,0 +1,37 @@
+template: ocpp
+products:
+  - description:
+      de: OCPP v1.6 kompatible Wallbox
+      en: OCPP v1.6 compatible Wallbox
+group: generic
```

Comment:
Wir sollten noch erwähnen, das in der Box eingestellt werden muss um sich mit evcc zu verbinden (ws://....).